src/nxt_socket.c

```diff
@@ -358,3 +370,29 @@ nxt_socket_error_level(nxt_err_t err)
         return NXT_LOG_ALERT;
     }
 }
+
+
+nxt_int_t
+nxt_socket_release_by_path(nxt_file_name_t *name)
+{
+    nxt_file_t          file;
+    nxt_file_info_t     fi;
+    nxt_int_t           ret;
+
+    (void) nxt_memzero(&file, sizeof(file));
```

Comment:
Redundant `(void)
```
#define                                                                       \
nxt_memzero(buf, length)                                                      \
    (void) memset(buf, 0, length)
```

Reply:
I'm doing this for readability, and I've seen a lot of this style of code in an already existing Unit. I think the compiler can easily handle this by removing it at the time of transpilation without increasing the size of the final program. But I won't do that anymore. Thank you for explaining this to me.

https://github.com/nginx/unit/blob/8138d15f76b1017ba61273f69416ffbe028b0d67/src/nxt_string.h#L45